### DISCUSSION TITLE
How to add custom notes?

### DISCUSSION THREAD

**Discussion by libracrumbs** [KadeArchive/Kade-Engine#1766] [Q&A] [answered]
I want to add custom notes like red halo notes from tricky mod, black and violet notes from bobs onslaught and sphagetti notes from weegee mod, does anyone know the code? (I am on KE 1.6 because its the latest version with the Icongrid.)

**Comment by EcoVeni** [comment, marked as answer]
watch this tutorial: https://www.youtube.com/watch?v=iiQfjYcpJfQ
